.emacs.d/test/lisp/run-tests.el

```diff
@@ -4,4 +4,17 @@
 
 (load-file (concat (file-name-directory load-file-name) "/dot-tests.el"))
 
+(when (require 'undercover nil 'noerror)
+  ;; undercover can't search for matching files recursively
+  ;; override a local function that fixes this
+  (advice-add
+   #'undercover--wildcards-to-files :override
+   (lambda (wildcards)
```

Comment:
We don't use lexical binding in our code, so there's no way for emacs to know if a variable is unused. That said, it probably would be good style to put the underscore anyway.